com.unity.render-pipelines.universal/Runtime/XR/XRPass.cs

```diff
@@ -46,6 +47,13 @@ internal XRView(Matrix4x4 proj, Matrix4x4 view, Rect vp, int dstSlice)
             viewport = vp;
             occlusionMesh = null;
             textureArraySlice = dstSlice;
+
+            var projectionParameters = proj.decomposeProjection;
+            float left = Math.Abs(projectionParameters.left);
+            float right = Math.Abs(projectionParameters.right);
+            float top = Math.Abs(projectionParameters.top);
+            float bottom = Math.Abs(projectionParameters.bottom);
+            eyeCenterUV = new Vector2(left / (right + left), top / (top + bottom));
```

Comment:
Can we add a method to avoid code duplication : computeEyeCenterUV(Matrix4x4 proj) ?

Reply:
Yes, we sure can !